pom.xml

```diff
@@ -119,6 +125,14 @@
 			<scope>provided</scope>
 		</dependency>
 
+                <!-- misc dependencies -->
+                <dependency>
+                    <groupId>org.projectlombok</groupId>
+                    <artifactId>lombok</artifactId>
+                    <version>1.18.12</version>
+                    <scope>provided</scope>
+                </dependency>
```

Comment:
Please remove this dependency.

Reply:
Lombok is not a dependency that 'leaks' into user apps.
I am disappointed that this is only coming up now because I discussed Lombok in my original email.

To me, this is like asking 
"I like OmniFaces, but please remove Java and JSF depencies
because they are \<bloated/slow/dying/ insert your favorite reason here\>"
Lombok is mainly here for the builder pattern, and in turn to make `static final transient` fields work in a serializable class. I know of no other clean method of getting that done.
I also can / will not redesign / rewrite this code from scratch to avoid a "dependency" I just don't have time for that.

Reply:
see https://stackoverflow.com/questions/2968876/final-transient-fields-and-serialization
the answer with readResolve() and `@Builder` used for copy-constructor